dmscripts/upload_dos_opportunities_email_list.py

```diff
@@ -26,19 +22,16 @@ def main(data_api_client, mailchimp_client, lot_data, logger):
         framework=lot_data["framework_slug"], lot=lot_data["lot_slug"]
     )
 
-    emails = find_user_emails(supplier_users, framework_services)
+    emails = frozenset(map(lambda x: x.lower(), find_user_emails(supplier_users, framework_services)))
```

Comment:
`frozenset(x.lower() for x in find_user_emails(supplier_users, framework_services))` is a little neater.

Reply:
👍